### PR TITLE
[IMP] doc: Rewrite mobile documentation

### PR DESCRIPTION
**Pull request by Yenthe666**
Description of the issue/feature this PR addresses: Documentation readability

Current behavior before PR: The mobile documentation was missing a lot of words and was either written fast or by somebody who's English wasn't very well (no offense)

Desired behavior after PR is merged: The mobile documentation has been brushed up. The random caps in sentences are removed and the sentences are fluid now. 
Minor fixes: typo's, caps, points, ...

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Yenthe666** [comment]
@mart-e as the previous doc fixes got me going I decided to dive into another documentation page. This one had quite some improvements that could be done. Can you check? Any feedback or things I should change further?

File: doc/reference/mobile.rst [hunk 1]
```diff
@@ -27,44 +26,44 @@ access all available mobile features.
 How does it work? 
 =================
 
-Internal working of Mobile application:
+Internal working of the mobile application:
```
Comment (Elkasitu) [inline doc/reference/mobile.rst]: workings? since it's being used as a noun

File: doc/reference/mobile.rst [hunk 2]
```diff
@@ -27,44 +26,44 @@ access all available mobile features.
...
 
 Process time taken by the Native component depends on what you are requesting
-from Native resources. Like Camera or GPS Location.
+from the Native resources. For example the Camera or GPS Location.
 
 How to use it?
 ==============
 
-As Odoo Web Framework, Mobile api can be used anywhere by getting object of
+Just like the Odoo Web Framework the Mobile API can be used anywhere by getting the object from
```
Comment (Elkasitu) [inline doc/reference/mobile.rst]: the comma was good imo

File: doc/reference/mobile.rst [hunk 3]
```diff
@@ -105,11 +104,11 @@ Show snackbar with action
...
 
 	:param object args: (*required*) **Message** to show in snackbar and action **button label** in Snackbar (optional)
-	:returns:  ``True`` if User click on Action button, ``False`` if SnackBar auto dismissed after some time
+	:returns:  ``True`` if the user clicks on the Action button, ``False`` if SnackBar auto dismissed after some time.
 
 Snackbars provide lightweight feedback about an operation. They show a brief
-message at the bottom of the screen on mobile and lower left on larger devices.
-Snackbars appear above all other elements on screen and only one can be
+message at the bottom of the screen on mobile or in the lower left corner on larger devices.
+Snackbars appears above all other elements on the screen and only one can be
```
Comment (Elkasitu) [inline doc/reference/mobile.rst]: appears is used for singular subjects, appear for plural

**Comment by Yenthe666** [comment]
@Elkasitu thanks a lot for the review! I've corrected the ones you suggested 😄 
I actually didn't know about appear / appears.

**Pull request merged**